catwatch-backend/src/main/resources/application.properties

```diff
@@ -19,7 +19,24 @@ spring.jpa.hibernate.ddl-auto=update
 # the scheduler runs at 8:01 AM every day
 schedule=0 1 8 * * *
 
-scoring.project=function(project) { return project.forksCount > 0 ? ( project.starsCount + project.forksCount + project.contributorsCount + project.commitsCount / 100 ) : 0 }
+scoring.project=function(project) {\
+                    var daysSinceLastPush = 0;\
+                    if (project.lastPushed) {\
+                        var year = project.lastPushed.match(/.*(....)$/)[1];\
+                        var month = project.lastPushed.match(/^... (...)/)[1];\
+                        var day = project.lastPushed.match(/^... ... (..)/)[1];\
+                        var lastPushedDate = new Date(day + ' ' + month + ' ' + year);\
```

Comment:
In this case month consists of letters, e.g. JAN. I think switching 12 cases would make it too verbose.